apps/backend/src/common/services/audit_service.py

```
@@ -0,0 +1,292 @@
+"""Audit logging service for security events."""
+
+import json
+import logging
+from datetime import datetime
+from typing import Dict, Any, Optional
+from enum import Enum
+
+from src.core.config import settings
```

Ruff (0.11.9)

`src.core.config.settings` imported but unused

Remove unused import: `src.core.config.settings`

(F401)